src/renderer/components/AgentBase.vue

```diff
@@ -16,9 +16,11 @@
             :value="did">
           </el-option>
         </el-select>
-        <!-- <el-select v-if="false" v-model="active_ledger_selector.ledger" filterable placeholder="activate ledger" >
+        <!-- <el-select  v-if="$refs.ledgerTab" 
```

Comment:
If possible, I think naming our components simply "Ledger" is a cleaner convention, especially considering that we intend to discontinue the use of tabs altogether.

Reply:
Minor enough that I won't let this keep from merging; we can come back with more appropriate naming when necessary.